xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/fees/FeeUtils.java

```diff
@@ -0,0 +1,90 @@
+package org.xrpl.xrpl4j.model.client.fees;
+
+import com.google.common.primitives.UnsignedLong;
+import org.xrpl.xrpl4j.model.ledger.SignerListObject;
+import org.xrpl.xrpl4j.model.transactions.XrpCurrencyAmount;
+
+import java.util.Objects;
+
+public class FeeUtils {
+
+  /**
+   * Computes the fee necessary for a multisigned transaction.
+   *
+   * <p>The transaction cost of a multisigned transaction must be at least {@code (N + 1) * (the normal
+   * transaction cost)}, where {@code N} is the number of signatures provided.
+   *
+   * @param currentLedgerFeeDrops The current ledger fee, represented as an {@link XrpCurrencyAmount}.
+   * @param signerList            The {@link SignerListObject} containing the signers of the transaction.
+   * @return An {@link XrpCurrencyAmount} representing the multisig fee.
+   */
+  public static XrpCurrencyAmount computeMultiSigFee(
+    final XrpCurrencyAmount currentLedgerFeeDrops,
+    final SignerListObject signerList
+  ) {
+    Objects.requireNonNull(currentLedgerFeeDrops);
+    Objects.requireNonNull(signerList);
+
+    return currentLedgerFeeDrops
+      .times(XrpCurrencyAmount.of(UnsignedLong.valueOf(signerList.signerEntries().size() + 1)));
+  }
+
+
+  /**
+   * Get value of fee to be used for submitting a transaction on the ledger. The value is calculated
+   * depending on the load on the job queue.
+   *
+   * @param feeResult {@link FeeResult} object received from XrplClient#fee() rippled call.
+   * @return {@link XrpCurrencyAmount} value of the fee that should be used for the transaction.
+   */
+  public static XrpCurrencyAmount calculateFeeDynamically(FeeResult feeResult) {
+    int currentQueueSize = feeResult.currentQueueSize().intValue();
+    int maxQueueSize = feeResult.maxQueueSize().get().intValue();
+    int queuePct = currentQueueSize / maxQueueSize;
```

Comment:
It's almost always better to use full words in variable names, so this should be `queuePercentage` I think